### ISSUE TITLE
Import issue while installing for Android

### ISSUE THREAD

**Issue by ArnaudRinquin** [Buglife/react-native-buglife#6]
Small issue that was an easy fix for me: the imports in my main Application file where not right. I needed to change from:

<img width="419" alt="screen shot 2018-01-24 at 22 53 07" src="https://user-images.githubusercontent.com/228886/35375039-57c3b710-015a-11e8-8beb-f3efba164fad.png">

to:

<img width="356" alt="screen shot 2018-01-24 at 22 54 08" src="https://user-images.githubusercontent.com/228886/35375044-5b95e2dc-015a-11e8-8f57-cf1ee56d3e66.png">

**Comment by schukin** [comment]
Thanks @ArnaudRinquin! I believe that old `import` in your main Application file was from an older version of react-native-buglife? (i.e. the version we helped you install, which wasn't quite ready for primetime ;) )

This _shouldn't_ be an issue for anyone doing a fresh install of react-native-buglife, since the import wouldn't be there in the first place... But yeah, thanks for bringing this up, since it's useful for any others affected by it. 👍 👍 

**tldr; For anyone previously using a version of react-native-buglife lower than v0.1.8, you may need to update imports in any .java files referencing Buglife classes.**
